x509-identity-mgmt/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dojot/x509-identity-mgmt",
-    "version": "1.0.0",
+    "version": "1.1.0a",
```

Comment:
What's the idea here? Will we bump the services' version each release?

Reply:
I only changed because the services are different from what is in the dev in relation to the code .. But I think there is no need